### ISSUE TITLE
Config file format is inconvenient when you have a lot of untyped dependencies

### ISSUE THREAD

**Issue by Phlogistique** [python/mypy#8918]
* Are you reporting a bug, or opening a feature request?

Feature request.

* Please insert below the code you are checking with mypy,
  or a mock-up repro if the source is private. We would appreciate
  if you try to simplify your case to a minimal repro.

When a project with many dependencies is ported to mypy, it will typically have to tell mypy to ignore the imports of these many dependencies. That means that the `mypy.ini` config file will look like this

```
[mypy-numpy]
ignore_missing_imports = True
[mypy-cv2]
ignore_missing_imports = True
[mypy-bson,bson.*]
ignore_missing_imports = True
```

For big projects, this list will be very long.

* What is the behavior/output you expect?

It would be nice if you could specify the list as a single option in the mypy config file:
```
[mypy]
ignore_missing_imports =
    numpy
    cv2
    bson
    ...
```

**Comment by JukkaL** [comment]
Yeah I can see how the current situation can be clumsy. The name of the setting should perhaps not be `ignore_missing_imports`, however, since that already has a meaning.

**Comment by Phlogistique** [comment]
How about `ignore_missing_imports_from`? Having a similar name makes it more discoverable, and `from` makes it clear that it is more restrictive than a blanket `ignore_missing_imports`.

**Comment by JukkaL** [comment]
Hmm I think that is too close to the existing configuration option. 

What about `allow_missing_stubs = numpy, cv2, ...`? This would communicate that the import might be available (in site-packages), but it's missing stubs. Also it would server as a hint about how to resolve the issue: install or create stubs.

**Comment by Phlogistique** [comment]
It sounds good to me.

It would require documenting the relationship between this and `ignore_missing_imports`.
